release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2049,15 +1957,7 @@ If a new node is being added to a Machine Config Pool that includes SR-IOV, this
 
 * The `stalld` service triggers a bug in the kernel, which results in the node freezing. In order to work around this issue, the Performance Addon Operator disables `stalld` by default. The fix impacts latency associated with DPDK based workloads, however the functionality will be restored once the kernel bug (link:https://bugzilla.redhat.com/show_bug.cgi?id=1912118[*BZ#1912118*]) is fixed.
 
-* Fluentd pods with the `ruby-kafka-1.1.0` and `fluent-plugin-kafka-0.13.1` gems are not compatible with Apache Kafka version 0.10.1.0.
-+
-As a result, log forwarding to Kafka fails with a message: `error_class=Kafka::DeliveryFailed error="Failed to send messages to flux-openshift-v4/1"`
-+
-The `ruby-kafka-0.7` gem dropped support for Kafka 0.10 in favor of native support for Kafka 0.11. The `ruby-kafka-1.0.0` gem added support for Kafka 2.3 and 2.4. The current version of OpenShift Logging tests and therefore supports Kafka version 2.4.1.
-+
-To work around this issue, upgrade to a supported version of Apache Kafka.
-+
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1907370[*BZ#1907370*])
+* Fluentd pods with the `ruby-kafka-1.1.0` and `fluent-plugin-kafka-0.13.1` gems are not compatible with Apache Kafka version 0.10.1.0. For more information, see xref:../logging/cluster-logging-release-notes.adoc[Red Hat OpenShift Logging 5.0 release notes]
```

Comment:
Do we want to link this to Known Issues? https://deploy-preview-29785--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html#openshift-logging-5-0-known-issues

Reply:
It seems like the RNs prefer using `xref:` to link to docs and `link:` to link to bz's.